### PR TITLE
Add SUL masthead and navbars

### LINKED ISSUE #23: Add Stanford Libraries header to layout
Feedback link split off to #42 

Replace header provided by `quick_search-generic_theme` with Stanford branded header
image

### PR DESCRIPTION
**Pull request by camillevilla**
Closes #23 

This PR adds:
- dummy navbar for Login / My Account / Feedback
- Stanford Libraries logo masthead that links to bento homepage
- Library services menu
- Containers and a `layout.scss` file
- View tests for the stacked header

### XL width
results page at XL width

### XS width
results page at XS width

### CODE REVIEW

**Comment by camillevilla** [comment]
@jkeck @jvine do users actually need to be able to access login / my account for Bento? I have some dummy links up there for now.

**Comment by camillevilla** [comment]
Ok, per our discussion in Slack, the Login link has been removed.

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	#subnavbar, #sul-footer, #masthead {
+    width: 970px;
+  }
+}
+
+@media (min-width: 1200px) {
+	#subnavbar, #sul-footer, #masthead {
+    width: 1170px;
+  }
+}
```
Comment (jkeck) [inline app/assets/stylesheets/layout.scss]: It seems like there is some odd alignment going on in this file.

File: app/assets/stylesheets/masthead.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#masthead-container {
+  background-color: #9D2520;
```
Comment (jkeck) [inline app/assets/stylesheets/masthead.scss]: Wondering if we should create our own variables file to store colors like this (if not already available to us).

File: app/assets/stylesheets/masthead.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  min-height: 100px;
+
+  #masthead {
+    margin: 0 auto;
+    height: 100px;
+    color: white;
```
Comment (jkeck) [inline app/assets/stylesheets/masthead.scss]: We should prob. use the `$white` variable here.

File: app/views/shared/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<div id="top-navbar-container">
+  <div id="top-navbar">
+    <ul>
+      <li><a href="https://library.stanford.edu/myaccount">My Account</a></li>
+      <li><a href='#'>Feedback</a></li>
```
Comment (jkeck) [inline app/views/shared/_header.html.erb]: @jvine is there somewhere we should send this link in the interim?

**Comment by jkeck** [comment]
In addition to the above comments, I noticed a little issue w/ the link (feedback, library services) alignment.  I think we may need some containers...but not entirely sure.  Totally fine to push that into a separate issue though.

link-alignment

File: app/assets/stylesheets/masthead.scss (hunk 2 above)
Comment (camillevilla) [reply app/assets/stylesheets/masthead.scss]: @jvine can you clarify which red I should use for the Bento app? 
Mockups have #9D2520, but that didn't map to any brand colors or pre-existing colors in SeachWorks. Should I be using Cardinal Red or Bright Red? 

stanford_reds

Comment (camillevilla) [reply app/assets/stylesheets/masthead.scss]: Let me know about the following areas: 
(1) font color for the Search bar
(2) background color for the masthead
(3) background color for bento section border-tops

File: app/views/shared/_header.html.erb (hunk 4 above)
Comment (jvine) [reply app/views/shared/_header.html.erb]: @jkeck you mean with no feedback form hooked up? there's always this:
http://library.stanford.edu/ask/email/feedback
but probably moot by now.

File: app/assets/stylesheets/masthead.scss (hunk 2 above)
Comment (jvine) [reply app/assets/stylesheets/masthead.scss]: @camillevilla Grrr omnigraffle doesn't seem to export colours accurately. The red is Cardinal throughout. Everywhere Cardinal. "Search" heading, border-tops, and the logo banner are Cardinal.

"Masthead" has a specific meaning in SW Catalog that I think is different here. 
- page background is #fbfbf9
- logo banner is #8c1515
- subnav menu bar is #797268

Does that cover it?

Comment (camillevilla) [reply app/assets/stylesheets/masthead.scss]: Yes! Thank you~

**Comment by camillevilla** [comment]
Alrighty @jkeck , requested changes have been made:
- moved colors to SCSS variables 
- confirmed bento colors with @jvine
- fixed indent issues in `layout.scss`
- added container to topnav links

**Pull request merged**